package.json

```diff
@@ -40,13 +39,31 @@
   "homepage": "https://github.com/denis-sokolov/remove-github-forks",
   "devDependencies": {
     "ava": "^3.0.0",
-    "eslint": "^6.8.0",
-    "prettier": "^1.18.2"
+    "xo": "^0.33.1"
   },
   "ava": {
     "files": [
       "test/**/*",
       "!test/lib"
     ]
+  },
+  "xo": {
+    "extends": [
+      "plugin:promise/recommended"
+    ],
+    "prettier": true,
+    "space": 2,
+    "rules": {
+      "camelcase": "off",
+      "unicorn/filename-case": [
+        "error",
+        {
+          "case": "camelCase"
+        }
+      ]
+    }
+  },
+  "prettier": {
+    "singleQuote": false
```

Comment:
Would you mind if we dropped all style configs and stick with the defaults? The less configuration, the better.

Reply:
Sure! I added these xo/prettier config to avoid too much thrashing, but if you're ok with tabs and single quotes, that's perfectly fine by me 👌

Reply:
Done: 681fd3e

Reply:
The good part of these tools is that they have good defaults, so I'm definitely ok with _less config_